nixos/modules/services/system/earlyoom.nix

```diff
@@ -63,6 +63,15 @@ in
           Enable debugging messages.
         '';
       };
+
+      notificationsCommand = mkOption {
+        type = types.string;
```

Comment:
Use `types.str`

Reply:
Yeah, `types.string` concatenates strings for merging, which is almost never what you want, so that's why it's deprecated.